lib/galaxy/datatypes/binary.py

```diff
@@ -42,11 +41,8 @@ def register_sniffable_binary_format(data_type, ext, type_class):
 
     @staticmethod
     def register_unsniffable_binary_ext(ext):
-        Binary.unsniffable_binary_formats.append(ext.lower())
-
-    @staticmethod
-    def is_ext_unsniffable(ext):
-        return ext in Binary.unsniffable_binary_formats
+        """Deprecated method."""
+        pass
```

Comment:
I'm sure there's a reason, but why leave this in and just 'pass'?  If it's here for compatibility, are we worried about external consumers receiving a `None`?  And should it maybe log "Hey, this code is a deprecated function and may not behave how you expect" or the like?

Reply:
I think you're reading the diff wrong, `is_ext_unsniffable()` has been removed, `register_unsniffable_binary_ext()` is kept for backward compatibility, but never returned anything.

Reply:
@nsoranzo D'oh, yes, I glanced through this too quickly.